hack/godep-save.sh

```diff
@@ -34,8 +34,8 @@ REQUIRED_BINS=(
 )
 
 pushd "${KUBE_ROOT}" > /dev/null
-  "${GODEP}" version
-  GO15VENDOREXPERIMENT=1 ${GODEP} save "${REQUIRED_BINS[@]}"
+  GOPATH=${GOPATH}:${KUBE_ROOT}/staging godep save "${REQUIRED_BINS[@]}"
```

Comment:
We should probably have an utility similarly to `kube::util::ensure_godep_version` to run the one we care about.